.github/workflows/comment-build-test.yml

```
@@ -0,0 +1,64 @@
+        run: |
+          comment="${{ github.event.comment.body }}"
+          if [[ "$comment" =~ /run-ci[[:space:]]+([^[:space:]]+) ]]; then
+            runner="${BASH_REMATCH[1]}"
+          else
+            runner="nixos-amd"
+          fi
+          echo "runner=$runner" >> $GITHUB_OUTPUT
+          echo "should_run=true" >> $GITHUB_OUTPUT
+
+      - name: React to comment
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)